shader/lighting/tonemapping.frag

```diff
@@ -10,6 +10,32 @@
 
 #include "upscale/lanczos.glsl"
 
+#if defined(COMPUTE)
+const uint THREADGROUP_SIZE = 8;
```

Comment:
Any reason to have a constant for `THREADGROUP_SIZE`? `gl_WorkGroupSize` should be just fine.